docs/content/concepts/authorization/authorizers.md

```diff
@@ -276,9 +276,24 @@ The webhook will receive JSON-marshalled `SubjectAccessReview` objects, that (co
 }
 ```
 
-
     The extra field will contain the logical cluster _name_ (e.g. o43u2gh528rtfg721rg92), not the human-readable path. Webhooks need to resolve the name to a path themselves if necessary.
 
+### Authorizer order
+
+By default, the authorizers are evaluated in the following order:
+
+1. **Always Allow Groups Authorizer**
+2. **Always Allow Paths Authorizer**
+3. **RBAC Chain**
+4. **Webhook Authorizer**
```

Comment:
Can we document how those translate to fields in `--authorizer-order` later on? so e.g.

```suggestion
1. **Always Allow Groups Authorizer** (`AlwaysAllowGroups`)
2. **Always Allow Paths Authorizer** (`AlwaysAllowPaths`)
3. **RBAC Chain** (`RBAC`)
4. **Webhook Authorizer** (`Webhook`)
```